versioned_docs/version-2.0.0/keploy-explained/debugger-guide.md

```diff
@@ -0,0 +1,77 @@
+---
+id: debugger-guide
+title: Debugger Guide
+sidebar_label: Debugger Guide
+tags:
+  - explanation
+  - dev guide
+  - debugger guide
+---
+
+# Debugger Guide 👨‍🔧
+
+In this blog, we will learn how to debug keploy using Visual Studio Code integrated debugger.  
+
+## Setting Up Configuration File
+
+To bring up the **Run and Debug** view, select the **Run and Debug** icon in the **Activity Bar** on the side of VS Code. You can also use the keyboard shortcut `Cmd+Shift+D`.  
```

Comment:
as I said something for both windows and macos (ctrl + shift +D and cmd+shift+D ) both